discordbot/requirements.txt

```diff
@@ -0,0 +1,236 @@
+aiohttp==3.7.4.post0; python_version >= "3.6" and python_full_version >= "3.5.3"
```

Comment:
What if we had here only the ones that we need on top of the default requirements.txt?

I.e. aiohttp, discord, discord_components, pymgur?

Reply:
My only worry would be if some of the other dependencies are also different versions, and if in the future we update other dependencies in the main requirements.txt, but that interferes with the discord.py. This discord.py package will be a continual pain since it is archived.

Reply:
Sounds good to me, I agree!